packages/react-components/src/components/MessageThread.tsx

```diff
@@ -966,6 +973,7 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
             showMessageStatus={showMessageStatus}
             messageStatus={messageProps.message.status}
             onActionButtonClick={onActionButtonClickMemo}
+            customizedFormatTimestampForChatMessage={customizedFormatTimestampForChatMessage}
```

Comment:
should it be an optionalProp called

dateTimeFormat? would we use it for anything other than chat messages?

Reply:
updated